### PR TITLE
Add @maridjuice

### PR DESCRIPTION
**Pull request by maridjuice**

### CODE REVIEW

**Comment by wooorm** [comment]
Does this code run? And show a bar chart?

You’re the first person to do this assignment, but there should be two more errors in the js!

**Comment by maridjuice** [comment]
@wooorm it does run. I see a chart and there are no errors in the console..?
screen shot 2017-10-02 at 23 07 23

**Comment by wooorm** [comment]
Fuck.

**Comment by maridjuice** [comment]
@wooorm Oops

**Comment by wooorm** [comment]
You’re right. I thought I made this assignment harder... 🤷‍♂️

**Comment by wooorm** [comment]
Other than the things in the CSS, great to go!

**Comment by maridjuice** [comment]
@wooorm Is this one of them?
screen shot 2017-10-02 at 23 13 00

I've also adjusted the css now. Will do a new pull request in a sec

**Comment by wooorm** [comment]
Good!

P.S. you can edit files here on GH, by clicking on the pencil on the top-right of a file.

File: site/class-2-debug/maridjuice/index.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    margin: 0;
+}
+
+.axis path,
+.axis line,
+.dot,
+rect {
+    stroke: black;
+    stroke-width: 2;
+}
```
Comment (wooorm) [inline site/class-2-debug/maridjuice/index.css]: Wait, you’re changing all the styles!!!

Comment (maridjuice) [reply site/class-2-debug/maridjuice/index.css]: Oops!

**Pull request merged**